src/ucs/sys/compiler_def.h

```diff
@@ -28,7 +28,7 @@
      switch(0) {case 0:case (_cond):;}
 
 /* Maximal allocation size for on-stack buffers */
-#define UCS_ALLOCA_MAX_SIZE  1200
+#define UCS_ALLOCA_MAX_SIZE  4096
```

Comment:
maybe move to dynamic alloc?
check if given above changes it's still big effort to check append success